src/g3log/time.hpp

```diff
@@ -60,7 +60,10 @@ namespace g3 {
    inline system_time_point to_system_time(const high_resolution_time_point& ts)
    {
 	   using namespace std::chrono;
-	   return time_point_cast<system_clock::duration>(system_clock::now() + (ts - high_resolution_clock::now()));
+	   static auto hrs_now = high_resolution_clock::now();
+	   static auto sys_now = system_clock::now();
+
+	   return time_point_cast<system_clock::duration>(sys_now + (ts - hrs_now));
```

Comment:
Please add a comment explaining the modification also as the reasoning why you time reduction happens is not very obvious.